### ISSUE TITLE
Nightly Build: Many Warnings in StructuralMechanics

### ISSUE THREAD

**Issue by philbucher** [KratosMultiphysics/Kratos#5003]
from the nightly build of CLANG:
~~~
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::VonMisesYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::VonMisesYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::VonMisesYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::VonMisesYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::VonMisesYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::VonMisesYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::VonMisesYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::VonMisesYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::ModifiedMohrCoulombYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::ModifiedMohrCoulombYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::ModifiedMohrCoulombYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::ModifiedMohrCoulombYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::ModifiedMohrCoulombYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::ModifiedMohrCoulombYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::ModifiedMohrCoulombYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::ModifiedMohrCoulombYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::TrescaYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::TrescaYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::TrescaYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::TrescaYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::TrescaYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::TrescaYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::TrescaYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::TrescaYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::DruckerPragerYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::DruckerPragerYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::DruckerPragerYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::DruckerPragerYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::DruckerPragerYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::DruckerPragerYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::DruckerPragerYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::DruckerPragerYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::RankineYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::RankineYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::RankineYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::RankineYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::RankineYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::RankineYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::RankineYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::RankineYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::SimoJuYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::SimoJuYieldSurface<Kratos::VonMisesPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::SimoJuYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::SimoJuYieldSurface<Kratos::ModifiedMohrCoulombPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::SimoJuYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::SimoJuYieldSurface<Kratos::DruckerPragerPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:175:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::SimoJuYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::Has' hides overloaded virtual functions [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h:213:10: warning: 'Kratos::GenericSmallStrainHighCycleFatigueLaw<Kratos::GenericConstitutiveLawIntegratorDamage<Kratos::SimoJuYieldSurface<Kratos::TrescaPlasticPotential<6> > > >::GetValue' hides overloaded virtual functions [-Woverloaded-virtual]
~~~

can you guys please have a look?
Thanks

**Comment by loumalouomega** [comment]
That belong to @SergioJimenezReyes

**Comment by AlejandroCornejo** [comment]
What is the meaning of "hides overloaded virtual functions"? We will solve this ofc

**Comment by loumalouomega** [comment]
That has methods with the same name as some virtual methods on the base class but with different arguments

**Comment by SergioJimenezReyes** [comment]
I paste here the GetValue and Has methods that seem to be the problem:
    double& GetValue(
        const Variable<double>& rThisVariable, 
        double& rValue) override;
    int& GetValue(
        const Variable<int>& rThisVariable, 
        int& rValue) override;
    bool Has(const Variable<double>& rThisVariable) override;
    bool Has(const Variable<int>& rThisVariable) override;

I have checked the format of these methods in the constitutive_law.cpp and it is the same

**Comment by philbucher** [comment]
your functions seem to have a different signature

**Comment by philbucher** [comment]
did you have a chance to look at the warnings already?

**Comment by SergioJimenezReyes** [comment]
I did some changes in the PR #5038 in this direction but I don't know if them solve the issue.

**Comment by philbucher** [comment]
no the warnings are still there

**Comment by SergioJimenezReyes** [comment]
I've done new changes in this direction. Hope to solve this in the next PR.

**Comment by RiccardoRossi** [comment]
is this still relevant?

**Comment by philbucher** [comment]
I think so, at least I don't remember it being solved
Also I haven't received a build from clang in a looong time, which usually indicates that there are many warnings

**Comment by philbucher** [comment]
There actually was a report tonight, warnings are still there 
Ping @SergioJimenezReyes

**Comment by SergioJimenezReyes** [comment]
I think that I did some changes regarding this a month ago and I included it in a commit, but I had no time for a PR, sorry. I'll try to do it during next week

**Comment by philbucher** [comment]
ping @SergioJimenezReyes is this solved?

**Comment by SergioJimenezReyes** [comment]
> ping @SergioJimenezReyes is this solved?

I will shortly do a PR with the changes. Sorry for the delay but I have been really busy.

**Comment by RiccardoRossi** [comment]
@AlejandroCornejo @SergioJimenezReyes @marandra ,  the warnings of the CI are really annoying. It should be relatively fast to fix them, at least the Has stuff. We @KratosMultiphysics/technical-committee  would really appreciate if you can take a look soon.

**Comment by SergioJimenezReyes** [comment]
I will try to do a PR with this during tomorrow. Sorry

**Comment by philbucher** [comment]
fixed in #8214
